### DISCUSSION TITLE
There is no active file

### DISCUSSION THREAD

**Discussion by gocerk** [Chia-Network/chia-blockchain#6351] [Support]
Hello, I'm trying to setup chia blockchain to my Ubuntu 20.04 Os. 

I'm just writing `sh install.sh` and its starting to install. Output: 

`
apt-get is /usr/bin/apt-get
Installing on Ubuntu 20.04 LTS.
Hit:1 http://tr.archive.ubuntu.com/ubuntu focal InRelease
Hit:2 http://tr.archive.ubuntu.com/ubuntu focal-updates InRelease              
Hit:3 http://tr.archive.ubuntu.com/ubuntu focal-backports InRelease            
Hit:4 http://ppa.launchpad.net/bablu-boy/nutty/ubuntu focal InRelease          
Hit:5 http://dl.google.com/linux/chrome/deb stable InRelease                   
Hit:6 http://packages.microsoft.com/repos/code stable InRelease                
Hit:7 https://swupdate.openvpn.net/community/openvpn3/repos focal InRelease    
Hit:8 http://repository.spotify.com stable InRelease                           
Ign:9 http://ppa.launchpad.net/elementary-os/stable/ubuntu focal InRelease     
Hit:10 http://security.ubuntu.com/ubuntu focal-security InRelease              
Hit:11 https://repo.steampowered.com/steam stable InRelease                    
Hit:12 https://deb.nodesource.com/node_14.x focal InRelease                    
Hit:13 http://ppa.launchpad.net/lutris-team/lutris/ubuntu focal InRelease      
Ign:14 https://storage.googleapis.com/download.dartlang.org/linux/debian stable InRelease
Hit:15 https://packages.microsoft.com/ubuntu/20.04/prod focal InRelease        
Hit:16 http://ppa.launchpad.net/maarten-fonville/android-studio/ubuntu focal InRelease
Hit:17 https://download.sublimetext.com apt/dev/ InRelease          
Hit:18 https://storage.googleapis.com/download.dartlang.org/linux/debian stable Release
Hit:19 http://ppa.launchpad.net/pinta-maintainers/pinta-stable/ubuntu focal InRelease
Ign:20 http://ppa.launchpad.net/webupd8team/java/ubuntu focal InRelease
Err:21 http://ppa.launchpad.net/elementary-os/stable/ubuntu focal Release
  404  Not Found [IP: 91.189.95.85 80]
Err:22 http://ppa.launchpad.net/webupd8team/java/ubuntu focal Release
  404  Not Found [IP: 91.189.95.85 80]
Reading package lists... Done
N: Skipping acquire of configured file 'main/binary-i386/Packages' as repository 'https://swupdate.openvpn.net/community/openvpn3/repos focal InRelease' doesn't support architecture 'i386'
E: The repository 'http://ppa.launchpad.net/elementary-os/stable/ubuntu focal Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://ppa.launchpad.net/webupd8team/java/ubuntu focal Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
`

and i cant run the file called activate
`
.: no such file or directory: ./activate
`

![image](https://user-images.githubusercontent.com/65464708/120465987-3de05980-c3a7-11eb-8f5a-52c7ffeee413.png)

**Comment by lvcivs** [comment]
The error message shows that `apt` on your system cannot find a third-party repository and aborts. The installation of chia then cannot go on.

You need to sort out the problematic third party repo. It looks like it is the one from `webupd8team`.

**Comment by BrandtH22** [comment]
@gocerk , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
